### ISSUE TITLE
Error: {ClusterServiceClassExternalName:"azure-cosmosdb-mongo-account"} does not exist

### ISSUE THREAD

**Issue by desreela** [Azure/blackbelt-aks-hackfest#98]
I am facing an error while applying the yaml file with the Service Instance for the OSBA lab.  

Even after installing osba with --set modules.minStability=EXPERIMENTAL and having the class name as azure-cosmosdb-mongo-account, getting the following error:

=====================
kubectl apply -f heroes-cosmosdb.yaml

Error from server (Forbidden): error when creating "heroes-cosmosdb.yaml": serviceinstances.servicecatalog.k8s.io "heroes-cosmosdb-instance" is forbidden: ClusterServiceClass {ClusterServiceClassExternalName:"azure-cosmosdb-mongo-account"} does not exist, cannot figure out the default ClusterServicePlan.

And azure-cosmos-mongo-account is not listed in the 'svcat get classes' and the 'svcat get plans'   output too.  AKS cluster is v1.9.6.

**Comment by chzbrgr71** [comment]
Apparently the OSBA project went into "GA" mode and they disabled services that are experimental. In order to get this to work, we need to install the v0.11.0 version of the OSBA Helm chart. 

I'm not sure of the syntax for this offhand, but maybe someone in the group can figure it out here. I think you just add something like `--version 0.11.0` to the helm install for the broker.

**Comment by desreela** [comment]
Got the OSBA installed as 'open-service-broker-azure-0.11.0' via helm using the flag --version 0.11.0.
++++++++++++++
helm install azure/open-service-broker-azure --name osba --namespace osba   --set azure.subscriptionId=$AZURE_SUBSCRIPTION_ID   --set azure.tenantId=$AZURE_TENANT_ID   --set azure.clientId=$AZURE_CLIENT_ID   --set azure.clientSecret=$AZURE_CLIENT_SECRET   --version 0.11.0
++++++++++++++
**However the issue persists.** 
Not able to get the  azure-cosmos-mongo-account listed under classes. And getting the same error message (posted originally) while trying to apply the yaml file.  Tried on different AKS clusters in different subscriptions in both Cloudshell as well as CentOS Vm bash shell.

**Comment by jeremyrickard** [comment]
Hey @desreela, you'll also need to use the flag to enable experimental services.

Give this a try:

```console
helm install azure/open-service-broker-azure --name osba --namespace osba --set azure.subscriptionId=$AZURE_SUBSCRIPTION_ID --set azure.tenantId=$AZURE_TENANT_ID --set azure.clientId=$AZURE_CLIENT_ID --set azure.clientSecret=$AZURE_CLIENT_SECRET --version 0.11.0 --set modules.minStability=EXPERIMENTAL
```

Once that finishes, you should be able to get the CosmosDB Services:

```console
vcat get plans --class azure-cosmosdb-sql
   NAME           CLASS                   DESCRIPTION                             UUID
+---------+--------------------+--------------------------------+--------------------------------------+
  sql-api   azure-cosmosdb-sql   Azure CosmosDB With SQL          58d7223d-934e-4fb5-a046-0c67781eb24e
                                 API (Database Account and
                                 Database)
```

**Comment by desreela** [comment]
@jeremyrickard, It worked now after including both the flags !!! 

--version 0.11.0 --set modules.minStability=EXPERIMENTAL

@chzbrgr71, @jeremyrickard Thanks a lot guys !!!

**Comment by mathieu-benoit** [comment]
FYI I added the mention of `--version 0.11.0` in the lab instructions via this PR #108, since that's the only way for now to be able to provision CosmosDB via OSBA. It's not working yet with `1.0.1` nor `1.1.0`.

**Comment by jeremyrickard** [comment]
I'll double check on this for you. With 1.1.0, you should be able to re-enable the experimental modules.

**Comment by mathieu-benoit** [comment]
FYI @chzbrgr71 or @jschluchter , this issue could be now closed since PR #108 has been merged into master.
